### DISCUSSION TITLE
CMD+W open new tab but not new window occasionally

### DISCUSSION THREAD

**Discussion by hilaolu** [raphamorim/rio#745] [Q&A]
I am using MacOS and rio 0.1.17. Pressing CMD+W sometimes open a new tab on current focus but not a window. The frequency is quite low and the condition to reproduce is elusive. I try many times but only confirm that it happen in 0.1.17 and branch main, without finding any clues. Both CollapsedTab and NativeTab has the problem.  Is it a feature and work as expected?

**Comment by raphamorim** [comment]
This is actually a bug I am fixing for next version, will convert this to issue

**Comment by raphamorim** [comment]
Closing this discussion and move to issue #756
